src/Rings/groebner.jl

```diff
@@ -162,12 +167,34 @@ function groebner_basis_with_transform(B::BiPolyArray; ord::Symbol = :degrevlex,
   return BiPolyArray(B.Ox, i), map_entries(x->B.Ox(x), m)
 end
 
-function lift_groebner_basis(I::MPolyIdeal, ord::Symbol; complete_reduction::Bool=false)
+
+@doc Markdown.doc"""
+    groebner_basis_with_transform(I::MPolyIdeal; ord::Symbol = :degrevlex, complete_reduction::Bool=false)
+
+Given an ideal `I` and optional parameters `ord` for a monomial ordering and `complete_reduction`
+this function computes a Groebner basis (if `complete_reduction = true` the reduced Groebner basis) of the
+ideal spanned by the elements in `B` w.r.t. the given monomial ordering `ord` and the
+transformation matrix from the ideal to the Groebner basis.
+
+###### Example
+```jldoctest
+julia> R,(x,y) = PolynomialRing(QQ,["x","y"])
+(Multivariate Polynomial Ring in x, y over Rational Field, fmpq_mpoly[x, y])
+
+julia> I = ideal([x*y^2-1,x^3+y^2+x*y])
+ideal generated by: x*y^2 - 1, x^3 + x*y + y^2
+
+julia> G,m = groebner_basis_with_transform(I)
+(fmpq_mpoly[x*y^2 - 1, x^3 + x*y + y^2, x^2 + y^4 + y], [1 0; 0 1; -x^2 - y y^2])
+```
+"""
+function groebner_basis_with_transform(I::MPolyIdeal; ord::Symbol = :degrevlex, complete_reduction::Bool=false)
```

Comment:
Hrrmm, Codecov still complains that this function is not covered by tests -- which tells me that we are probably not executing the jldoctests as part of our CI, despite what I thought... I'll file an issue to remind us to look into it.